tests/functional/build-remote-trustless-after.sh

```diff
@@ -1,2 +1,7 @@
-outPath=$(readlink -f $TEST_ROOT/result)
-grep 'FOO BAR BAZ' ${remoteDir}/${outPath}
+# shellcheck shell=bash
+
+# Variables must be defined by script that sources
```

Comment:
```suggestion
# Variables must be defined by script that sources, so
```

Wasn't sure whether the next line was connected.

```suggestion
# Variables must be defined by caller, so
```

`source` feels like an implementation detail to me, and reads weird, but you decide.